### PR TITLE
Feature: #73, generate api with model prompt, support html generate

### PR DESCRIPTION
**Pull request by nutsjian**
### What is the purpose of the change
1. support text input box generate html
2. generate api with model prompt

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible 
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: solidui-web/install.sh [hunk 1]
```diff
@@ -120,31 +120,31 @@ soliduiConf(){
 
 				location /solidui/models {
 					proxy_pass $solidui_url_models; #Solidui backend address
-					proxy_set_header Host $host;
```
Comment (dlimeng) [inline solidui-web/install.sh]: by default

**Comment by nutsjian** [comment]
generate html code:

image

image

**Comment by dlimeng** [comment]
LGTM

File: solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql [hunk 2]
```diff
@@ -180,8 +181,10 @@ INSERT INTO `solidui_datasource_type` (`id`, `name`, `description`, `option`, `c
 VALUES
 (1,'mysql','mysql','mysql','mysql','mysql',3);
 
-INSERT INTO `solidui_model_type` (`id`, `name`, `type_name`, `prompt`, `token`, `baseurl`)
+INSERT INTO `solidui_model_type` (`id`, `name`, `code`, `type_name`, `prompt`, `token`, `baseurl`)
 VALUES
-(1,'gpt-3.5-turbo','gpt',NULL,NULL,NULL),
-(2,'gpt-4','gpt',NULL,NULL,NULL),
-(3,'chatglm_lite','chatglm',NULL,NULL,NULL);
+(1,'gpt-3.5-turbo', 'code', 'gpt','First, here is a history of what I asked you to do earlier. The actual prompt follows after ENDOFHISTORY. History:\n\n{}ENDOFHISTORY.\n\nWrite Python code that does the following: \n\n{}\n\nNote, the code is going to be executed in a Jupyter Python kernel.\n\nLast instruction, and this is the most important, just return code. No other outputs, as your full response will directly be executed in the kernel. \n\nTeacher mode: if you want to give a download link, just print it as <a href=''/solidui/models/download?file=INSERT_FILENAME_HERE''>Download file</a>. Replace INSERT_FILENAME_HERE with the actual filename. So just print that HTML to stdout. No actual downloading of files!',NULL,NULL),
```
Comment (dlimeng) [inline solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql]: code replaces python

File: solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql [hunk 3]
```diff
@@ -180,8 +181,10 @@ INSERT INTO `solidui_datasource_type` (`id`, `name`, `description`, `option`, `c
...
+(2,'gpt-4','code', 'gpt',NULL,NULL,NULL),
```
Comment (dlimeng) [inline solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql]: code replaces python

File: solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql [hunk 4]
```diff
@@ -180,8 +181,10 @@ INSERT INTO `solidui_datasource_type` (`id`, `name`, `description`, `option`, `c
...
+(3,'chatglm_lite','code', 'chatglm',NULL,NULL,NULL),
```
Comment (dlimeng) [inline solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql]: code replaces python ,  prompt is not set

Comment (nutsjian) [reply solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql]: done

File: solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql (hunk 3 above)
Comment (nutsjian) [reply solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql]: done

File: solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql (hunk 2 above)
Comment (nutsjian) [reply solidui-entrance/src/main/resources/sql/mysql/solidui_mysql.sql]: done

**Comment by dlimeng** [comment]
LGTM

**Pull request merged**